src/combine-results.ts

```diff
@@ -1,11 +1,26 @@
 import fs from "fs";
 import path from "path";
 
-import * as artifact from "@actions/artifact";
-import * as core from "@actions/core";
-import * as exec from "@actions/exec";
-import * as github from "@actions/github";
-import * as io from "@actions/io";
+import artifact from "@actions/artifact";
+import core from "@actions/core";
+import exec from "@actions/exec";
+import github from "@actions/github";
+import io from "@actions/io";
+
+const formatBody = (query: string) => `# Query
```

Comment:
I just thought this was cleaner, as I kind of hate template type strings in function bodies - as it interrupts reading the logic.

Reply:
Yep, definitely better!